app/services/activities/import_from_csv.rb

```diff
@@ -82,12 +84,20 @@ def add_error(row_number, column, value, message)
     class ActivityUpdater
       attr_reader :errors, :activity, :row
 
-      def initialize(row, organisation)
+      def initialize(row:, user:, delivery_partner_organisation:)
```

Comment:
How about renaming `user` to `uploader` here as well. Handy to differentiate in cases (BEIS uploading?) where the uploader might not belong to the delivery partner organisation..?